src/app/story/story.component.ts

```diff
@@ -79,10 +82,18 @@ export class StoryComponent implements OnInit, AfterViewInit, OnDestroy {
         const nodeDisplayNamePipe = new NodeDisplayNamePipe();
         const titleDisplay = nodeDisplayNamePipe.transform(this.selectedNode.display_name);
         this.titleService.setTitle(titleDisplay);
+        this.processingTimeFlag = story.meta.processing_time_flag;
+        this.getToast();
       }
     );
   }
 
+  getToast() {
```

Comment:
This function is only called in one place and it's only three lines, so it doesn't need to be a function. I think it's easier to read in this case if there's no function. Could you please replace the function call with the code in the function body?